core/types.go

```diff
@@ -72,13 +73,14 @@ func (d DutyType) String() string {
 		DutySyncMessage:             "sync_message",
 		DutyPrepareSyncContribution: "prepare_sync_contribution",
 		DutySyncContribution:        "sync_contribution",
+		DutyInfoSync:                "info_sync",
```

Comment:
what is this duty for?

Reply:
we prioritise "cluster-agreed-upon-supported-version" every epoch. This is used to decide to upgrade or not.